diff --git a/pyproject.toml b/pyproject.toml
index 8065fa2..f5c21df 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -6,7 +6,7 @@ authors = [
     {name = "Cas van Tijn"}
 ]
 readme = "README.md"
-license = {file = "LICENSE"}
+license-files = ["LICENSE" ]
 requires-python = ">= 3.8"
 
 [tool.mypy]
@@ -31,3 +31,5 @@ experimental = true
 max_line_length = 80
 ignore = ["E124", "E125", "E126", "E128", "E261"]
 
+[tool.setuptools]
+py-modules = []
\ No newline at end of file
